plugins/inputs/snmp_trap/snmp_trap.go

```diff
@@ -15,11 +15,9 @@ import (
 	"github.com/gosnmp/gosnmp"
 )
 
-var defaultTimeout = config.Duration(time.Second * 5)
-
 type SnmpTrap struct {
 	ServiceAddress string          `toml:"service_address"`
-	Timeout        config.Duration `toml:"timeout"`
+	Timeout        config.Duration `toml:"timeout" deprecated:"1.20.0;unused option"`
```

Comment:
Should this be 1.22.0?

Reply:
It has become obsolete since 1.21, I don’t know

Reply:
The option became unused since @MyaLongmire switched translation to gosmi. I think you should ask her about how to proceed.

Reply:
I think @powersj had a good idea. It should be mentioned in the Readme in a minor release

Reply:
Great, will try something, but won’t be before next week.

Reply:
I think we are doing a release tomorrow. I'll a pr later this afternoon with a note in the readme

Reply:
[This](https://github.com/influxdata/telegraf/pull/10373) pull request adds the deprecation notice in the readme